.packit.yaml

```
@@ -74,6 +74,12 @@ jobs:
     project: packit-stable
     list_on_homepage: True
     preserve_project: True
+    actions:
+      create-archive:
+        - "python3 setup.py sdist --dist-dir ."
+        - "sh -c 'echo packitos-$(python3 setup.py --version).tar.gz'"
+      get-current-version:
+        - bash -c "echo $(python3 setup.py --version).stable"
```

I don't know, wouldn't something like this be better, since we know that the `.dev*` suffix indicates a pre-release version?

```suggestion
        - bash -c 'v=$(python3 setup.py --version) && echo ${v/.dev/~dev}'
```